Add support for WiFi site surveys (#619)
NordicSemiconductor/asset-tracker-cloud-docs: docs/app/WiFiGeolocation.rst

.. _app-wifigeolocation:

> WiFi geolocation

Reviewer: ```suggestion
Wi-Fi geolocation
```

> ################

Reviewer: ```suggestion
#################
```

.. contents::
   :local:
   :depth: 2

> If wi-fi site survey geolocation (ground fix service) is implemented, the devices can be shown on the map based on the location of their nearest wi-fi access points.

Reviewer: ```suggestion
If Wi-Fi site survey geolocation (ground fix service) is implemented, the devices can be shown on the map based on the location of the Wi-Fi access points nearest to them.
```

Third-party location APIs
*************************

nRF Asset Tracker implements the following third-party location APIs:

>  - nRF Cloud Location Services

Reviewer: ```suggestion
* nRF Cloud Location Services
```

> These APIs enable calculation of the rough location of a device as soon as it sends the nearest wi-fi access points information to the cloud.

Reviewer: ```suggestion
These APIs enable calculation of the rough location of a device as soon as it sends the information for the nearest Wi-Fi access points to the cloud.
```

> Follow the configuration guide in the respective implementation to enable nRF Cloud's group fix API:

Reviewer: Should this be ground fix, not group fix?

Reviewer 2: I believe Ground Fix API is the right thing, note also the capitalization because that is how it is in the API docs.
